### DISCUSSION TITLE
Sparse neural learned models on ES

### DISCUSSION THREAD

**Discussion by cadurosar** [castorini/pyserini#1242] [General]
Hey, I just saw the new test of HNSW using ES (https://github.com/castorini/pyserini/pull/1241), pretty nice stuff! Would it be possible to compare with sparse neural learned models (so unicoil/splade/deepimpact etc)? Thinking mostly because this could fix one of the problems I have on comparing dense and sparse (different software optimizing for different things), even if it would still be dependent on the underlying implementations at least it would be on the same software.

**Comment by lintool** [comment]
Yup! Switching to Slack for secret musings ;)
